### ISSUE TITLE
Readme Documentation for playLocal example incorrect

### ISSUE THREAD

**Issue by richk1** [luanpotter/audioplayer#17]
The playLocal() example in README.md is exactly the same as the play() example. It should be:
     <code>final result = await audioPlayer.play(localFilePath, isLocal: true);</code>
I realize this is an exact copy from the original source, which is still also in error. Not sure who I should be correcting, but I think this is the repo I expect to use, so.... :)

**Comment by luanpotter** [comment]
You're absolutely correct, thanks for letting me know. Would you like to make a PR fixing the README? Or I can do it myself, no problems either way :)

**Comment by richk1** [comment]
I'm kindof a noob with git - haven't done a pull request yet. So probably best if you just take care of it. Thanks!
